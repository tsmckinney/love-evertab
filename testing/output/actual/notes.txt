# Actual Graphics Output
The images generated by the tests