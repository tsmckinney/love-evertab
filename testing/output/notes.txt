# Testing Output
Any tests run will output an XML, MD, and HTML file here, assuming the tests are run with readwrite permissions for this repo