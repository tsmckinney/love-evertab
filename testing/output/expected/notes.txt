# Expected Graphics Output
The images expected by the tests